tornado_swagger/builders.py

```diff
@@ -57,6 +57,10 @@ def extract_swagger_docs(endpoint_doc):
             "description": "Swagger document could not be loaded from docstring",
             "tags": ["Invalid Swagger"],
         }
+        error = str(e)
```

Comment:
that  **e** var is not defind, this is the reason the test error.

Reply:
Yes, you're right... I made the code change locally and then copy-pasted for the pull request but neglected that. Sorry about that.

The reason the error works better in the tags is because it's multi-line. It's much harder to read the error when jammed into the description string.

Reply:
I'll provide an example later today.

Reply:
This is how the exception appears in the generated endpoint JSON if you put its multiple lines into the tags field:

![image](https://user-images.githubusercontent.com/6498506/80727332-d2043100-8ac2-11ea-8abe-42f7dbfe8d2c.png)

If we just stick the exception into the description, it ends up being much harder to read:

![image](https://user-images.githubusercontent.com/6498506/80727548-1099eb80-8ac3-11ea-9dae-560e84886fa8.png)

Either way is an improvement, of course.

Reply:
Fixed the lack of an e declaration, by the way.